src/pages/signin/PasswordForm.js

```diff
@@ -57,12 +57,19 @@ class PasswordForm extends React.Component {
      */
     validateAndSubmitForm() {
         if (!this.state.password.trim()
-            || (this.props.account.requiresTwoFactorAuth && !this.state.twoFactorAuthCode.trim())
+            && (this.props.account.requiresTwoFactorAuth && !this.state.twoFactorAuthCode.trim())
         ) {
             this.setState({formError: 'passwordForm.pleaseFillOutAllFields'});
             return;
         }
-
+        if (!this.state.password.trim()) {
```

Comment:
Not strictly required by our style guide, but I think it'd be cleaner if there was a line break between each conditional. That seems to be how it is more frequently done in our codebase from what I can see.

Reply:
Added line break after each `if statement`

Reply:
Sweet, thanks.